src/background.c

```diff
@@ -647,10 +647,11 @@ background_monitor (gpointer data)
     {
       g_main_context_iteration (monitor_context, TRUE);
       /* We check twice, to avoid killing unlucky apps hit at a bad time */
-      sleep (30);
+      sleep (5);
       check_background_apps ();
-      sleep (30);
+      sleep (10);
```

Comment:
So we check, sleep 10s, check again. That means you went from 30 to 10 seconds grace period until killing, if you're unlucky?

Reply:
That *would* have been the case, if we didn't change the policy together. This PR makes the background portal more lenient with these apps, and paired with https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/merge_requests/56, we only kill apps explicitly forbidden from running in background.

Reply:
That's a good point; it allows us to be more agressive when checking.